From 24c0532dcefe5d52daef61840d0e671f739d2623 Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@foss.st.com>
Date: Fri, 20 Jan 2023 15:37:04 +0100
Subject: [PATCH 257/954] feat(stm32mp2-fdts): add oem_fip_enc_key mapping

Add the DT node for the OTP fuses used to store the secret used to
derive TF-A FIP binaries decryption key.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 04d2feeb90f30bbd68294b526c5e20b3dd210cbb feat(stm32mp2-fdts): add oem_fip_enc_key mapping]
Signed-off-by: Yann Gautier <yann.gautier@foss.st.com>
Change-Id: Ie3fa84c013ca2aa31233543ce30302f84dd85752
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/286094
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Lionel DEBIEVE <lionel.debieve@foss.st.com>
---
 fdts/stm32mp251.dtsi | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/fdts/stm32mp251.dtsi b/fdts/stm32mp251.dtsi
index 87891a6440..2fc364c196 100644
--- a/fdts/stm32mp251.dtsi
+++ b/fdts/stm32mp251.dtsi
@@ -317,6 +317,9 @@
 			pkh_otp: otp144@240 {
 				reg = <0x240 0x20>;
 			};
+			oem_fip_enc_key: otp260@410 {
+				reg = <0x410 0x20>;
+			};
 		};
 
 		iwdg1: watchdog@44010000 {
-- 
2.53.0

